thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomalydetection/alertFilterAutotune/AlertFilterAutotuneFactory.java

```diff
@@ -0,0 +1,68 @@
+package com.linkedin.thirdeye.anomalydetection.alertFilterAutotune;
```

Comment:
The way this class is coded, looks more like you want to have a Factory for 'AlertFilter' not 'AlertFilterAutotune'. Since we do not need to store 'autotune method' in anomalyFunction specs.